docs/user/artifacts/function_list

```diff
@@ -1014,6 +1014,15 @@ isometric(geometry, extrusion): isometric
 
 * isometric: Geometry 1:1 required
 
+lapply(source, expression): result
```

Comment:
These are generated from the code example here: 

https://github.com/geotools/geotools/blob/master/docs/src/main/java/org/geotools/main/FilterExamples.java#L527